tests/zeus/vcs/conftest.py

```diff
@@ -1,29 +1,50 @@
 import pytest
 
+from contextlib import asynccontextmanager
 from uuid import UUID, uuid4
 
 from zeus.vcs.server import build_app
 
 
+class FakePool:
+    def __init__(self, pool):
+        self.pool = pool
+        self.conn = None
+
+    @asynccontextmanager
+    async def acquire(self):
+        if self.conn is None:
+            ctx = self.pool.acquire()
+            self.conn = await ctx.__aenter__()
+        yield self.conn
+
+
 @pytest.fixture(scope="function")
-async def vcs_app(loop):
+async def vcs_app(loop, req_ctx):
```

Comment:
flask will throw a complicated error if we dont load some context (when accessing certain parts of current_app)